dev-workflow-ui/src/ch/ivyteam/workflowui/NotesIvyDevWfBean.java

```diff
@@ -0,0 +1,32 @@
+package ch.ivyteam.workflowui;
+
+import java.util.List;
+
+import javax.faces.bean.ManagedBean;
+import javax.faces.bean.ViewScoped;
+
+import ch.ivyteam.ivy.workflow.INote;
+import ch.ivyteam.workflowui.util.TaskUtil;
+
+@ManagedBean
+@ViewScoped
+public class NotesIvyDevWfBean {
```

Comment:
there is no need anymore to use "IvyDevWf". dev-workflow-ui is running a web-jar and if somebody installs it as iar, he/she could install it in it's own application.